pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -223,6 +227,31 @@ def test_resolution(self):
         assert dt.as_unit("ms").resolution == Timedelta(milliseconds=1)
         assert dt.as_unit("s").resolution == Timedelta(seconds=1)
 
+    @pytest.mark.parametrize(
+        "date_string, expected_true, expected_false",
+        [
+            ("0000-2-29", 1, 2),
```

Comment:
can we include a negative date too?

Reply:
Ah for BCE right. Sorry I miss understood what you meant by negative before.  

My current implementation of Gauss's isn't happy with negative dates, will work on fixing that.

Reply:
I'm having a hard time finding a valid source for expected values to confirm that my implementation of Gauss's algorithm is performing as expected for negative dates.

I tried the built-in `datetime` lib to compare against' but it didn't' seem to like negative date-strings. 

Is the a source of truth that you'd recommend?

Reply:
I'd suggest polars:
```python
import polars as pl
pl.Series(['-2000-01-01']).str.to_date()
```
It's not an optional dependency (though perhaps we should consider it? that's for a separate issue anyway) so we can't use it in the parametric tests, but we can use it to generate expected values by hand for this unit test

Reply:
looks like you've done it correctly anyway 🙌

Reply:
Thanks @MarcoGorelli appreciate the tip re: Polars!